src/Components/FilterComponents/SelectFilter.js

```diff
@@ -98,7 +98,7 @@ function SelectFilter({
   }
 
   return (
-    <div className="mb-2" data-testid="field-container">
+    <div className="mb-2" data-testid="field-container" id="field-container">
```

Comment:
@tanvirchahal issue here is that `id` must be unique but you may have several Select Filters I guess

Reply:
@anuveyatsu refactored code to use `data-testid` and removed all new added IDs